### PR TITLE
Bundle updated

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR runs `bundle update`. Looks like JWT had a major upgrade, and `omniauth-artsy` had a patch _down_grade? Not sure about that – the [upgrade is here](https://github.com/artsy/omniauth-artsy/pull/10). It looks pushed to RubyGems.org, so I'm not sure.

### CODE REVIEW

**Comment by ashfurrow** [comment]
@ashkan18 I tried that, no luck. Let me see what happens if I specify in `Gemfile`.

**Comment by ashfurrow** [comment]
Really weird:

```diff
diff --git a/Gemfile b/Gemfile
index eb0e71c..68d688f 100644
--- a/Gemfile
+++ b/Gemfile
@@ -24,6 +24,7 @@ gem 'sentry-raven'
 gem 'sidekiq'
 gem 'stripe'
 gem 'taxjar-ruby', require: 'taxjar'
+gem 'omniauth-artsy', '>= 0.2.3'
 
 group :development, :test do
   gem 'byebug'
diff --git a/Gemfile.lock b/Gemfile.lock
index a12f63e..2502789 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -124,7 +124,7 @@ GEM
     erubi (1.7.1)
     execjs (2.7.0)
     fabrication (2.20.1)
-    faraday (0.15.3)
+    faraday (0.12.2)
       multipart-post (>= 1.2, < 3)
     faraday-http-cache (1.3.1)
       faraday (~> 0.8)
@@ -159,7 +159,6 @@ GEM
       domain_name (~> 0.5)
     http-form_data (1.0.3)
     http_parser.rb (0.6.0)
-    httpauth (0.2.1)
     i18n (1.1.0)
       concurrent-ruby (~> 1.0)
     inherited_resources (1.9.0)
@@ -173,7 +172,7 @@ GEM
       railties (>= 4.2.0)
       thor (>= 0.14, < 2.0)
     json (2.1.0)
-    jwt (2.1.0)
+    jwt (1.5.6)
     kaminari (1.1.1)
       activesupport (>= 4.1.0)
       kaminari-actionview (= 1.1.1)
@@ -211,26 +210,29 @@ GEM
       i18n (>= 0.6.4, <= 2)
     msgpack (1.2.4)
     multi_json (1.13.1)
+    multi_xml (0.6.0)
     multipart-post (2.0.0)
     nap (1.1.0)
     nio4r (2.3.1)
     no_proxy_fix (0.1.2)
     nokogiri (1.8.5)
       mini_portile2 (~> 2.3.0)
-    oauth2 (0.6.1)
-      faraday (~> 0.7)
-      httpauth (~> 0.1)
+    oauth2 (1.4.0)
+      faraday (>= 0.8, < 0.13)
+      jwt (~> 1.0)
       multi_json (~> 1.3)
+      multi_xml (~> 0.5)
+      rack (>= 1.2, < 3)
     octokit (4.12.0)
       sawyer (~> 0.8.0, >= 0.5.3)
     omniauth (1.8.1)
       hashie (>= 3.4.6, < 3.6.0)
       rack (>= 1.6.2, < 3)
-    omniauth-artsy (0.2.2)
-      omniauth-oauth2 (>= 1.0.2)
-    omniauth-oauth2 (1.0.2)
-      oauth2 (~> 0.6.0)
-      omniauth (~> 1.0)
+    omniauth-artsy (0.2.3)
+      omniauth-oauth2 (>= 1.1.2)
+    omniauth-oauth2 (1.5.0)
+      oauth2 (~> 1.1)
+      omniauth (~> 1.2)
     open4 (1.3.4)
     opentracing (0.4.1)
     paper_trail (10.0.1)
@@ -397,6 +399,7 @@ DEPENDENCIES
   listen
   micromachine
   money
+  omniauth-artsy (>= 0.2.3)
   paper_trail
   pg
   puma
```

I'll take a look.

**Comment by ashfurrow** [comment]
Looks like an issue with [omniauth-oauth2 1.2.0](https://rubygems.org/gems/omniauth-oauth2/versions/1.2.0). It has a cap on Faraday at 0.10.0, so I think Bundler downgraded omniauth-artsy so that it could get a more up-to-date omniauth-oauth2 and faraday versions. Really weird. I'll add a note in Gemfile lock that explains why we need to specify an omniauth-artsy version.

File: Gemfile [hunk 1]
```diff
@@ -19,6 +19,8 @@ gem 'graphql'
 gem 'jwt'
 gem 'micromachine'
 gem 'money' # Library for dealing with money and currency conversion
+# omniauth-artsy version specifier is required since otherwise Bundler will downgrade omniauth-artsy in order to upgrade omniauth-oauth2 and faraday. See https://github.com/artsy/exchange/pull/225#issuecomment-428999929 for more info.
```
Comment (jonallured) [inline Gemfile]: Nice one!

**Comment by ashkan18** [comment]
tricky stuff! thanks for looking @ashfurrow lets test on staging after this

**Comment by ashfurrow** [comment]
I just ran through the happy path, post-deploy, and it works fine 👍 How granular should one be when testing bundle updates like this?

**Pull request merged**